src/applications/vaos/referral-appointments/ReviewAndConfirm.unit.spec.jsx

```diff
@@ -225,17 +190,48 @@ describe('VAOS Component: ReviewAndConfirm', () => {
     await screen.findByTestId('continue-button');
     expect(screen.getByTestId('continue-button')).to.exist;
     await userEvent.click(screen.getByTestId('continue-button'));
-    // Wait for the postReferralAppointment call to complete
-    waitFor(() =>
-      expect(screen.getByTestId('continue-button')).to.have.attribute(
-        'loading',
-        'false',
-      ),
-    );
+
     await screen.findByTestId('create-error-alert');
     expect(screen.getByTestId('create-error-alert')).to.contain.text(
       'We couldn’t schedule this appointment',
     );
     sandbox.assert.calledOnce(requestStub);
   });
+  it('should fetch draft appointment info on mount if not in store', async () => {
```

Comment:
What do you think about adding `requestStub.withArgs` to this? We could probably use the same test for choose date and time.